doc/content.tex

```diff
@@ -1,1349 +0,0 @@
-
-    %%%%%%%%
-    \section{Releases}
-    By the time writing of this thesis there was only one release with version \texttt{0.1.Final}. It is also available
-    in the Maven central repository. Complete group id is \texttt{org.hawkular.datamining}. The most important
-    artifact ids are listed in table \ref{tab:datamining-modules}. Release notes are available on
-    Github\footnote{Available at \url{https://github.com/hawkular/hawkular-datamining/releases}.}.
-
-    %%%%%%%%
-    \section{Retrospective}
-    In this section it is mentioned how the project was changing during the implementation, what did not worked well and
-    what could have be done better.
-
-    At the beginning of the project Apache Spark\footnote{Open source cluster computing framework available at
-    \url{https://spark.apache.org/}.} was used for building regression models. Its machine learning library (MLIB)
-    offers several learning algorithms. In Data Mining linear regression with stochastic gradient descent was used.
-    Because the gradient is stochastic produced predictions are not very robust and quickly change by significant value.
-
-    Spark was mainly used because of its streaming extension that can handle
-    high-throughput stream data \cite{apache-spark}. There is also time series library
-    \texttt{spark-ts}\footnote{Available at \url{https://github.com/sryza/spark-timeseries}.} which contains some
-    time series models but does not offer functionality of automatic model selection.
-
-    Due to the problems with linear regression discussed in \ref{sec:linear-regression} and lack of automatic
-    forecaster decision was made to drop Spark and directly implement exponential smoothing models.
-
-    During the implementation a lot of time was spent on integration with other modules, in particular with Inventory.
-    The problem was that Inventory is developed by another party and any concrete document
-    describing the integration was not proposed. The design of the integration was understood differently
-    by both parties, what lead to reimplementing of some parts. Therefore it is better to precisely define the
-    integration interface with other modules or projects.
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Evaluation of Implemented Models} \label{chap:evaluation}
-The evaluation of the implemented models compare the results produced by models from language R.
-The used methodology and more comprehensive description of tested objectives is listed in section \ref{sec:methodology}.
-
-Models from language R were selected because they are widely used by forecasters all around the world and therefore
-for this work considered as referential. Concretely models are from the package
-\texttt{forecast}\footnote{Available at \url{https://cran.r-project.org/web/packages/forecast/index.html}.}.
-Function used for fitting
```

Comment:
Review of existing monitoring solutions and libraries for time series modelling follows.